docs/model_quality.mdx

```diff
@@ -0,0 +1,104 @@
+
+## Ad Platform
+
+An ad platform should provide 1) requests received, 2) requests sent (traffic shaping), and 3) emissions data from datacenters or cloud providers
+
+| Server emissions | Request data | Frequency | Model Quality |
+| ---------------- | ------------ | --------- | ------------- |
+| Not provided     | Not provided | N/a       | 1             |
+| Global           | Global       | Annual    | 2             |
+| Regional         | Regional     | Annual    | 3             |
+| Global           | Global       | Monthly   | 4             |
+| Regional         | Regional     | Monthly   | 5             |
+
+## Property
+
+For media properties, key metrics include session time, session weight, and ad load. For OOH screens, the key metric is power draw. A measured metric is acquired on a per-property basis - for instance, using Google Analytics on a website or a power meter on a screen. An estimated or average metric uses a third-party source or historical averages to project key metrics, for instance, using screen size and type to model power usage.
+
+| Property Metrics              | Geography | Model Quality   |
+| ----------------------------- | ----------- | ------------- |
+| Not available                 | N/a         | 1             |
+| Spend-based                   | N/a         | 2             |
+| Average or estimated activity | Global      | 3             |
+| Observed or measured activity | Global      | 4             |
+| Observed or measured activity | Regional    | 5             |
+
+## Ad Stack
+
+The accuracy of the ad stack used by the publisher for the placement depends on accurate representation of all direct and indirect vendors. The ad stack score starts at 1 and increments based on modeled components.
+
+| Ad Stack Component                        | Model Quality |
+| ----------------------------------------- | ------------- |
+| All direct vendors identified             | +1            |
+| All indirect vendors identified           | +1            |
+| 90% of direct vendors have quality > 1    | +1            |
+| 90% of direct vendors have quality > 3    | +1            |
+
+## Input granularity
+
+Provided inputs must match the underlying model in order to provide accurate output. For instance, if the user provides "ANZ" as a country, that might not match "AU" or "NZ", causing a mapping issue that could cause a fallback to using worldwide grid mix and a significant loss of accuracy.
+
+Granularity % is the percentage of input fields provided.
+
+| Granularity % | Model Quality |
+| ------------- | ------------- |
+| < 30%         | 1             |
+| 30% - 50%     | 2             |
+| 50% - 70%     | 3             |
+| 70% - 90%     | 4             |
+| 90% - 100%    | 5             |
+
+Example:
+
+| Field           | Provided Input | Match   |
+| --------------- | -------------- | ------- |
+| Channel         | web            | yes     |
```

Comment:
generally a given inventory identifier resolves to a single channel (and given the difficulty of mapping traditional reports to the channel column) I'd be hesitant to penalize not providing channel.

also, other input cols that I'd think we would want to consider when measuring input granularity score:
- country
- region (if relevant based on country)
- device type
- network